common/variant/src/main/java/org/apache/spark/types/variant/Variant.java

```diff
@@ -113,6 +126,11 @@ public String getString() {
     return VariantUtil.getString(value, pos);
   }
 
+  // Get the type info bits from a variant value.
+  public int getTypeInfo() {
```

Comment:
I can't easily tell, but what uses this api?

Reply:
It is used when throwing an `UNKNOWN_PRIMITIVE_TYPE_IN_VARIANT` exception from scopes which don't directly have access to the type info.